.changeset/cold-dots-talk.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/form-field': patch
+---
+
+- Adds `data-lgid` to `Label`, `Description`, `Error`, and `optional`.
+- Removes `role="presentation"` from `Checkmark` and `Warning` icons and replaces it with `aria-hidden`. The `title` attribute is omitted when using `role="presentation"` and we rely on `title` to query the DOM for those icons.
```

Comment:
@bruugey Here's the explanation for the `aria` change

Reply:
Huh, interesting!